content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -37,10 +35,10 @@ Other functionality is provided by the Agent binary directly. For example, `data
 | `sudo service datadog-agent`                      | `sudo datadog-agent --help`                            | Display command usage              |
 | `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent -- datadog-agent check <check_name>` | Run a check                        |
 
-**NB**: If `service` is not available on your system, use:
+**Note**: If `service` is not available on your system, use:
 
-* `upstart`-based systems: `sudo initctl start/stop/restart datadog-agent`
-* `systemd`-based systems: `sudo systemctl start/stop/restart datadog-agent`
```

Comment:
I think the reason we mentioned the full `systemctl/initctl start/stop/restart datadog-agent` command here is that the order between `start/stop/restart` (the command) and `datadog-agent` (the service name) is different from `service`, i.e.:

* upstart/systemd: `command service_name`
* service: `service_name command`

Reply:
I agree that documenting how to use OS-specific service managers is a bit out-of-scope of these docs, however many customers (and DD engineers btw) don't know much about how to use them so I think it's still worth including these details.